src/make-comment.ts

```diff
@@ -33,25 +33,23 @@ const makeComment = async (message: string, commentData: CommentData): Promise<v
     });
 
     const botComment = comments.data.find((comment) => comment.body?.includes('<!-- coverage-action-comment -->'));
-    const customMessage = getInput('customMessage');
 
     console.log(botComment);
-    console.log(message);
     console.log(commentData);
 
     if (!botComment) {
       octokit.issues.createComment({
         owner: owner,
         repo: repo,
         issue_number: pullRequestNumber,
-        body: customMessage === 'comment' ? message : await constructComment(commentData),
+        body: message ?? (await constructComment(commentData)),
```

Comment:
This check if message is message is defined and constructing another message should be done earlier on in the function, like at the very top. 

`const body = message ?? (await constructComment(commentData))`

Then just use body everywhere else